docs/DeveloperGuide.adoc

```diff
@@ -308,11 +308,21 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |student |add profile picture to added contact |know at a glance who is the person in my contact
 
-|`* * *` |student |edit profile picture to added contact |edit the picture if changes are necessary
+|`* * *` |student |delete profile picture to added contact |
 
-|`* * *` |student |delete profile picture to added contact|
+|`* * *` |student |add category specific remark |filter out a contact’s remark according to type of contact
```

Comment:
Missing the word "can" before filter. The header doesn't have "can". Same for some of your other user stories.